### ISSUE TITLE
aspp pooling

### ISSUE THREAD

**Issue by hx-0614** [WZMIAOMIAO/deep-learning-for-image-processing#608]
aspp pooling 这个结构报错 
我发现地一个自适应池化层会将特征图的大小变成1*1
但是1*1尺寸的特征图我各种测试都无法执行 batchnormal的操作请问如何解决

**Comment by WZMIAOMIAO** [comment]
aspp pooling后本来就没有bn层，为什么要加BN层？

**Comment by hx-0614** [comment]
谢谢已经解决，是aspp pooling里面的bn层，因为我测试是batchsize 1测试会bug 在train模式下，谢谢



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月30日(星期六) 晚上6:08
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [WZMIAOMIAO/deep-learning-for-image-processing] aspp pooling (Issue #608)





 
aspp pooling后本来就没有bn层，为什么要加BN层？

**Comment by WZMIAOMIAO** [comment]
deeplab v3是有bn，只要训练时bs不等于1，测试时model.eval()就行

**Comment by hx-0614** [comment]
是的我发现了这个问题，谢谢 感谢




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月30日(星期六) 晚上6:15
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [WZMIAOMIAO/deep-learning-for-image-processing] aspp pooling (Issue #608)





 
deeplab v3是有bn，只要训练时bs不等于1，测试时model.eval()就行
